### ISSUE TITLE
The (endless?) chmod saga

### ISSUE THREAD

**Issue by jaromil** [dyne/Tomb#295]
Also seen in issue #279 (but not related to the discussion there) tomb log outputs highlight a current problem we have with file permissions:
```
mount_tomb:135: permission denied: /media/test/.uid
mount_tomb:137: permission denied: /media/test/.tty
mount_tomb:140: permission denied: /media/test/.host
mount_tomb:144: permission denied: /media/test/.last
```

This is an old problem related to usage of `chmod` and related security concerns, but the problem may now be fixed by a recent refactoring of the `sudo` invocation. I'll try to patch tomb so that it does a `chmod` of the files to the current user, since this really makes it much more usable for a lot of people. The behaviour should be optionally switched off by the `-n` flag (dryrun)

**Comment by jaromil** [comment]
Now addressed in #297 - comments welcome

**Comment by jaromil** [comment]
done. we reverted to chowning tomb contents upon mount.
